docs/DIRECTORY_STRUCTURE.md

```diff
@@ -33,78 +36,92 @@ The following is a brief overview of the project's directory structure under the
+Stores all icon images (typically in SVG format) used in the project.
 
-### `components/icons/`
+### `assets/images/logos/`
 
-Houses icon components, primarily as SVG wrappers or components.
+Stores all logo images used in the project.
+
+### `components/`
+
+Stores non-feature-specific components.
+
+### `components/layouts/`
+
+Stores shared layout components composed of other components from the `parts/` directory.
 
 ### `components/pages/`
 
-Implements actual pages, which are then imported into 'page.tsx' files in the 'app/' directory.
+Stores page components that implement actual pages, which are then imported into 'page.tsx' files in the `app/` directory.
 
 ### `components/parts/`
 
-Features shared components that are larger than basic UI components, such as headers and footers.
+Stores generic parts, such as headers, footers, and navigation bars.
+
+### `components/typography/`
+
+<!-- https://ui.shadcn.com/docs/components/typography -->
+
+Stores shared typography components, such as headings, paragraphs, and lists. Recommended to use the components exampled in [shadcnUI](https://ui.shadcn.com/docs/components/typography).
 
 ### `components/ui/`
 
-Includes shared UI components, such as buttons, cards, text fields. Recommended to use [shadcnUI](https://ui.shadcn.com/docs) components.
+Stores shared UI components, such as buttons, cards, text fields. Recommended to use [shadcnUI](https://ui.shadcn.com/docs) components.
 
 ### `const/`
 
-Stores all constant values utilized in the project.
+Stores constants used across the project.
 
 ### `features/`
 
-Each application feature has a dedicated directory, following a feature-driven structure.
+Each application feature has a dedicated directory, following a "feature-driven structure". Inspired by [this article](https://dev.to/profydev/screaming-architecture-evolution-of-a-react-folder-structure-4g25#indexjs-as-public-api).
 
 ### `features/<feature_name>/`
 
-Directories specific to each feature.
-
-### `features/<feature_name>/api/`
-
-Contains API related files.
+Stores all directories and files related to a specific feature.
 
 ### `features/<feature_name>/components/`
 
-Houses non-shared components unique to the feature.
+Stores feature-specific components.
 
-### `features/<feature_name>/hooks/`
+### `features/<feature_name>/lib/`
 
-Includes non-shared custom hooks relevant to the feature.
+Stores feature-specific code modules or external libraries.
 
 ### `features/<feature_name>/types/`
 
-Stores non-shared types specific to the feature.
+Stores feature-specific types.
 
 ### `features/<feature_name>/utils/`
 
-Contains non-shared utility functions relevant to the feature.
+Stores feature-specific utility functions.
 
 ### `hooks/`
 
-Shared custom hooks utilized across various features.
+Stores all [custom hooks](https://react.dev/learn/reusing-logic-with-custom-hooks) used across the project.
 
 ### `lib/`
```

Comment:
Is it not needed to mention we store API calling file  in `lib` directory?

Reply:
Oh, absolutely! That was very important.